### PR TITLE
Add dotenv to prepare for deployment

### PR DESCRIPTION
**Pull request by LuigiMenez**

### CODE REVIEW

File: src/Components/ExtSensor/index.jsx [hunk 1]
```diff
@@ -1,11 +1,14 @@
 import axios from 'axios';
+import dotenv from 'dotenv';
 import { useState, useEffect } from 'react';
 import { Marker, Popup, useMap } from 'react-leaflet';
 
 export default function ExtSensor() {
   const [position, setPosition] = useState(null);
   const [closerSensor, setCloserSensor] = useState(null);
```
Comment (PierreGoe) [inline src/Components/ExtSensor/index.jsx:9]: remove Empty line

File: src/Components/ExtSensor/index.jsx [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
 
+  dotenv.config();
+
   const map = useMap();
```
Comment (PierreGoe) [inline src/Components/ExtSensor/index.jsx:13]: remove Empty line

File: src/Components/MarkerIndoor/index.jsx [hunk 3]
```diff
@@ -13,6 +14,8 @@ import SPopup from './style';
 import './index.css';
 import EqaiContext from '../../Context/EqaiContext';
```
Comment (PierreGoe) [inline src/Components/MarkerIndoor/index.jsx:16]: remove Empty line

File: src/Components/MarkerIndoor/index.jsx [hunk 4]
```diff
@@ -13,6 +14,8 @@ import SPopup from './style';
...
 
+dotenv.config();
+
```
Comment (PierreGoe) [inline src/Components/MarkerIndoor/index.jsx:18]: remove Empty line

File: src/Components/MarkerOutdoor/index.jsx [hunk 5]
```diff
@@ -12,6 +13,8 @@ import SPopup from './style';
 import './index.css';
 import EqaiContext from '../../Context/EqaiContext';
```
Comment (PierreGoe) [inline src/Components/MarkerOutdoor/index.jsx:16]: remove Empty line

File: src/Components/MarkerOutdoor/index.jsx [hunk 6]
```diff
@@ -12,6 +13,8 @@ import SPopup from './style';
...
 
+dotenv.config();
+
```
Comment (PierreGoe) [inline src/Components/MarkerOutdoor/index.jsx:18]: remove Empty line

**Review by PierreGoe — APPROVED** [review]

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**